src/Helper/Mapping.php

```diff
@@ -56,91 +56,360 @@ public static function delivery_type() {
 	}
 
 	/**
-	 * Product code mapping.
+	 * Products code & required options mapping.
 	 *
-	 * @return Array
+	 * @return array[]
 	 */
-	public static function product_code() {
+	public static function products_data() {
 		return array(
 			'NL' => array(
 				'NL'  => array(
 					'delivery_day'  => array(
-						'3085' => array(),
-						'3385' => array( 'only_home_address' ),
-						'3090' => array( 'return_no_answer' ),
-						'3087' => array( 'insured_shipping' ),
-						'3189' => array( 'signature_on_delivery' ),
-						'3390' => array( 'return_no_answer', 'only_home_address' ),
-						'3094' => array( 'insured_shipping', 'return_no_answer' ),
-						'3089' => array( 'signature_on_delivery', 'only_home_address' ),
-						'3389' => array( 'signature_on_delivery', 'return_no_answer' ),
-						'3096' => array( 'signature_on_delivery', 'only_home_address', 'return_no_answer' ),
-						'2928' => array( 'letterbox' ),
-						'3438' => array( 'id_check' ),
+						array(
+							'combination' => array(),
+							'code'        => '3085',
+							'options'     => array()
```

Comment:
```suggestion
							'options'     => array(),
```
Let's standardize the array based on WP standard. Can you try to add `,` for all last item in the array?